src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    public void handle (Storage storage, FruitTransaction transaction);
```

Comment:
According to Java interface design best practices, you don't need to declare interface methods as `public` because they are implicitly public. You can remove the `public` modifier.